### ISSUE TITLE
Page sidebar: Add clickable tooltip (question mark) next to "Action Plan" label with help text"

### ISSUE THREAD

**Issue by mjsamberg** [TheFridayInstitute/studentslead#32]
Suggested text: Throughout this course, you’ll have an opportunity to build an action plan. Your action plan will support you as you learn more about how you learn and how to share this knowledge with others.

**Comment by mjsamberg** [comment]
@adreier @christinasimmons A tooltip here isn't going to work. The question mark circle wraps at a variety of screen sizes because of the length of the words "My Action Plan". Also, the text is too long for a tooltip - I can do a dialog box, but that's about it. We can also shrink the text by about half and just put it right under the header.

**Comment by christinasimmons** [comment]
@mjsamberg @adreier I think it would be best to hide the text vs. putting it under the header. A clickable dialog box may be fine, but what would students click on to launch it if a question mark circle doesn't fit? (I also think we can trim that text.)

**Comment by adreier** [comment]
@mjsamberg @christinasimmons I don't think that putting it under the header will look right. This also speaks to the need to create an Orientation page that will help address some of these issues (Mark and I discussed this morning), and I think we're close to the point where we've made enough decisions to actually know what will go into it. Is there any sort of visual indicator that will work universally? Something that would trigger students to visit either an FAQ/Orientation/etc. to learn more about something?

**Comment by christinasimmons** [comment]
@mjsamberg @adreier Mark, can we get some clarification? Alex and I are chatting about this now. We agree we need point of use support. And a dialog box that you click vs. tool tip that you hover over is okay, though not as clear. But are you saying a question mark next to the head (or anything next to the head) will not fit? If so, what are options? (Personally I think it's okay if the tool tip wraps on various screens.)

**Comment by mjsamberg** [comment]
It is going to wrap. 

Also, the text is too long for a tool tip. If it can be shortened, a tool-tip will work fine.

**Comment by christinasimmons** [comment]
@mjsamberg @adreier When Alex and I discussed yesterday we were in agreement that clarity would be better than the visual ideal. (So, let's let it wrap.) And would this text fit? (Alex, let us know if you have a better version to suggest.)

Your responses throughout the course will help you to build an action plan you can take with you.

**Comment by adreier** [comment]
You will build an action plan throughout the course that you can take with you.
